From a01ab015f9c7da3342fd9f31c201cbdb34e8c5a9 Mon Sep 17 00:00:00 2001
From: sgourdas <jim31700@gmail.com>
Date: Wed, 20 Mar 2024 23:20:18 +0200
Subject: [PATCH] Fixing part of issue #988. Results were never shown in
 fulltext search. Instead a "No results were found ..." message was shown.

---
 src/urlschemehandler.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/urlschemehandler.cpp b/src/urlschemehandler.cpp
index 956c8840..0e3515cb 100644
--- a/src/urlschemehandler.cpp
+++ b/src/urlschemehandler.cpp
@@ -133,8 +133,8 @@ UrlSchemeHandler::handleSearchRequest(QWebEngineUrlRequestJob* request)
     }
     kiwix::SearchRenderer renderer(
         search->getResults(start, pageLength),
-        search->getEstimatedMatches(),
-        start);
+        start,
+        search->getEstimatedMatches());
     renderer.setSearchPattern(searchQuery);
     renderer.setSearchBookQuery("content="+bookId.toStdString());
     renderer.setProtocolPrefix("zim://");
